src/ResourceManager/Sql/Commands.Sql/Properties/Resources.Designer.cs

```diff
@@ -673,6 +673,15 @@ internal class Resources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to Warning: Cutting off access to the key &apos;{0}&apos; may result in data loss on this server. Learn about best practices at https://docs.microsoft.com/sql/relational-databases/security/encryption/transparent-data-encryption-byok-azure-sql. Do you want to proceed?.
```

Comment:
I updated this to an aka.ms link. Note that the link currently is routed to standard TDE documentation, but once TDE with BYOK is officially in public preview, the link will route to the proper documentation.